Products/ExternalEditor/ExternalEditor.py

```diff
@@ -16,8 +16,11 @@
 
 # Zope External Editor Product by Casey Duncan
 
-from string import join  # For Zope 2.3 compatibility
-import types
+try:
+    from string import join  # For Zope 2.3 compatibility
+except ImportError:
+    # Python 3
+    join = str.join
```

Comment:
I think this should work on Python 2, too. And the code using `join` could be rewritten to use `'delim'.join([arg1, arg2, …])`.

Reply:
Definitely... no need to define join. I will update the PR ASAP